cranelift-codegen/meta-python/base/instructions.py

```diff
@@ -764,6 +764,15 @@
         ins=(src, dst),
         other_side_effects=True)
 
+copy_nop = Instruction(
+        'copy_nop', r"""
+        Stack-slot-to-the-same-stack-slot copy, which is guaranteed to turn
```

Comment:
Can you add that it's intended to be internally-used only? (and mirror this change in the Rust code)